### PR TITLE
Inital homepage

### PR DESCRIPTION
**Pull request by Nesssu**
The initial homepage is now ready. 

To change between the admin user and non-admin user while testing, change the boolean "admin" from the "Home" function state (true -> admin & false -> non-admin).

### CODE REVIEW

**Comment by Nesssu** [comment]
[Implement the home page mockup](https://trello.com/c/Dah8fmr1/14-implement-the-home-page-mockup)

**Review by Jezerig — APPROVED** [review]

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,8 +1,426 @@
...
+
+  const [admin, setAdmin] = useState(props.admin)
+  const [details, setDetails] = useState("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse mollis imperdiet est, ut maximus est lobortis non. Aliquam bibendum venenatis mi, a auctor lacus interdum feugiat. Aenean nec leo a diam iaculis iaculis. Vestibulum cursus tincidunt neque, quis euismod dolor tincidunt ac.")
+  const [manageDetails, setmanageDetails] = useState(false)
+  const [detailsHistory, setDetailsHistory] = useState("")
+
+
+  return(
+    <div className='Details'>
+      <div className='Backround'>
```
Comment (jannejjj) [inline client/src/components/Home.js:35]: Typo :D

**Review by jannejjj — APPROVED** [review]
Wow, great job! THe only thing I thought was tha maybe the Modals could be separated into their own files but I don't think that's a huge problem at all.

+check the small typo

**Review by jannejjj — APPROVED** [review]

**Comment by jannejjj** [comment]
Nice! You can go ahead and merge this now

**Pull request merged**